src/DNA.cpp

```diff
@@ -489,3 +549,108 @@ string DNA::get_binary_representation(int value, int bit_count) {
 
     return output;
 }//end get_binary_representation
+
+/**
+ * rand_num
+ * 
+ * Returns a random integer [start, end).
+ **/
+int DNA::rand_num(int start, int end) {
```

Comment:
Is there a specific reason you chose to make your own random function instead of using a library function?

Reply:
It is a wrapper of the original, which provides a wider variety of functionality.